src/screens/SettingsScreen.js

```diff
@@ -0,0 +1,159 @@
+import PropTypes from 'prop-types';
+import React, { useContext, useState } from 'react';
+import { RefreshControl, SectionList, StyleSheet, TouchableOpacity, View } from 'react-native';
+
+import { NetworkContext } from '../NetworkProvider';
+import { OrientationContext } from '../OrientationProvider';
+import { GlobalSettingsContext } from '../GlobalSettingsProvider';
+import { colors, device, normalize, texts } from '../config';
+import {
+  CardListItem,
+  Icon,
+  RegularText,
+  SafeAreaViewFlex,
+  TextListItem,
+  Title,
+  TitleContainer,
+  TitleShadow,
+  Wrapper
+} from '../components';
+import { SettingsListItem } from '../components/SettingsListItem';
+import { arrowLeft } from '../icons';
+
+export const SettingsScreen = ({ navigation }) => {
+  const { isConnected, isMainserverUp } = useContext(NetworkContext);
+  const { orientation, dimensions } = useContext(OrientationContext);
+  const globalSettings = useContext(GlobalSettingsContext);
+  const [refreshing, setRefreshing] = useState(false);
+
+  const keyExtractor = (item, index) => `index${index}-id${item.id}`;
+
+  const renderSectionHeader = ({ section: { title } }) =>
+    !!title && (
+      <View>
+        <TitleContainer>
+          <Title accessibilityLabel={`${title} (Überschrift)`}>{title}</Title>
+        </TitleContainer>
+        {device.platform === 'ios' && <TitleShadow />}
+      </View>
+    );
+
+  const refresh = () => {
+    setRefreshing(true);
+    // TODO: do we need pull to refresh on the settings screen?
+    // we simulate state change of `refreshing` with setting it to `true` first and after
+    // a timeout to `false` again, which will result in a re-rendering of the screen.
+    setTimeout(() => {
+      setRefreshing(false);
+    }, 500);
+  };
+
+  // TODO: get data from global settings
+  const { sections = {} } = globalSettings;
+
+  const sectionedData = [
+    {
+      data: [
+        {
+          title: texts.settingsTitles.pushNotifications,
+          topDivider: true,
+          type: 'toggle',
+          value: false
```

Comment:
we need some action key here. maybe two, one for activation and another one for deactivation.

Reply:
`onActivate` und `onDeactivate`?

Reply:
done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/123/commits/e5b495d1d6ce00b77ec351138d14f095a96750eb

Reply:
names seem good for now.

if we feel they are inappropriate once we use them, we can change them accordingly